presto-cache/src/main/java/com/facebook/presto/cache/alluxio/AlluxioCacheConfig.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.cache.alluxio;
+
+import com.facebook.airlift.configuration.Config;
+
+public class AlluxioCacheConfig
+{
+    private boolean metricsCollectionEnabled = true;
```

Comment:
Can we add the config to disable the async functionality?

Reply:
Added. Please note the async write functionality is not added in Alluxio release version 2.2.0. The added config property will work with Alluxio 2.2.1 and above once released.

Reply:
I assume even with 2.2.0 it would not harm.

Reply:
yeah, it should work